android/tangram/src/com/mapzen/tangram/geometry/Geometry.java

```diff
@@ -0,0 +1,35 @@
+package com.mapzen.tangram.geometry;
+
+import java.util.Map;
+
+/**
+ * Geometry is an abstract container of LngLat points and associated properties
+ */
+public abstract class Geometry {
+
+    protected double[] coordinates;
+    protected int[] rings;
+    protected String[] properties;
```

Comment:
Does it make sense to add a serialID to this?

Could be useful in explicitly removing a specific geometry feature from a set of features of a MapData?